src/tactic/dependencies.lean

```diff
@@ -291,8 +291,8 @@ meta def hyps_depend_on_local_name_set (hs : list expr) (ns : name_set) :
       let go : expr → list bool × expr_set → tactic (list bool × expr_set) :=
       λ h ⟨deps, cache⟩, do {
         (h_dep, cache) ← hyp_depends_on_local_name_set' cache h ns,
-        pure (h_dep :: deps, cache)
-      } in
+        pure (h_dep :: deps, cache) }
+      in
```

Comment:
Here and in the other `tactic` files -- perhaps there are just different whitespace conventions?  The style here seems pretty systematic, and was presumably written by expert users.  @fpvandoorn @digama0

Reply:
These changes look good to me!